pysteps/__init__.py

```diff
@@ -21,6 +21,10 @@
 from . import verification as vf
 from . import visualization as plt
 
+logging.getLogger(__name__).addHandler(logging.NullHandler())
```

Comment:
Maybe we can define the default handler set at INFO level to be equivalent to the print statement. 
That is, no additional formatting. _logger.info(msg) equivalent to print(msg)

That will reduce the verbosity of the outputs.  

We can add additional information (like time, level, etc) for the DEBUG level. 

What do you think @fox91?

Reply:
I agree with @aperezhortal : it is a good idea to use the INFO level for all print statements that are now unconditionally executed, while DEBUG can be used for those print statements that are currently conditional to a verbose argument or similar (e.g. the measure_time argument in the nowcast.steps() method).

Reply:
I'm changing `DEBUG` to `INFO` in the next commit, thanks for the clarifications.